components/dashboard/src/workspaces/Workspaces.tsx

```diff
@@ -52,38 +44,18 @@ export default function () {
     const showStartWSModal = () => setIsTemplateModelOpen(true);
     const hideStartWSModal = () => setIsTemplateModelOpen(false);
 
-    const getRecentSuggestions: () => WsStartEntry[] = () => {
-        if (workspaceModel) {
-            const all = workspaceModel.getAllFetchedWorkspaces();
-            if (all && all.size > 0) {
-                const index = new Map<string, WsStartEntry & { lastUse: string }>();
-                for (const ws of Array.from(all.values())) {
-                    const repoUrl = Workspace.getFullRepositoryUrl(ws.workspace);
-                    if (repoUrl) {
-                        const lastUse = WorkspaceInfo.lastActiveISODate(ws);
-                        let entry = index.get(repoUrl);
-                        if (!entry) {
-                            entry = {
-                                title: Workspace.getFullRepositoryName(ws.workspace) || repoUrl,
-                                description: repoUrl,
-                                startUrl: gitpodHostUrl.withContext(repoUrl).toString(),
-                                lastUse,
-                            };
-                            index.set(repoUrl, entry);
-                        } else {
-                            if (entry.lastUse.localeCompare(lastUse) < 0) {
-                                entry.lastUse = lastUse;
-                            }
-                        }
-                    }
-                }
-                const list = Array.from(index.values());
-                list.sort((a, b) => b.lastUse.localeCompare(a.lastUse));
-                return list;
+    useEffect(() => {
+        const onKeyDown = (event: KeyboardEvent) => {
+            if (['k', 'o', 'p'].includes(event.key) && (event.metaKey || event.ctrlKey)) {
```

Comment:
**suggestion**: I'd suggest to go with only one shortcut here to avoid hurting memory muscle in the long run or confuse users. I'd vote for keeping only `CMD+K` for this one.

Reply:
+1

Reply:
Thanks! I'd personally vote for using `Cmd+P` and also `Ctrl+P` (for non Mac users).

Also, are there really downsides to having an "official" keystroke (that we advertise everywhere) + also "secretly" catching the other two keystrokes, and doing something useful with them instead of nothing?

We could always rebind these to other features in the future, but I'm a big fan of going to where our users already are, instead of trying to force/"educate" everyone into a single behavior.

Reply:
For some more context, the concepts of the **command palette** and **quick switcher** overlap with what we're trying to achieve here with this modal. Some applications merge these two while other keep them distinctly separate.💡 

> I'd personally vote for using Cmd+P and also Ctrl+P (for non Mac users).

For historical context, `CMD/Ctrl+P` was predominantly the first key binding mapped for triggering search or command palettes while `CMD/Ctrl+K` recently started being used for command palettes[[1](https://philipcdavis.com/writing/command-palette-interfaces)][[2](https://blog.superhuman.com/how-to-build-a-remarkable-command-palette/)][[3](https://dribbble.com/tags/command_palette)][...]. FWIW, `CMD/Ctrl+O` has been historically used to open a file in many applications or operating systems[[4](https://en.wikipedia.org/wiki/Keyboard_shortcut#%22Sacred%22_keybindings)][[5](https://support.microsoft.com/en-us/topic/keyboard-shortcuts-for-the-file-menu-in-office-for-windows-1c4b07af-be2e-4c19-8d33-c600a8904043#bkmk_open)][...]. ⚔️ 

> .. are there really downsides to having an ...

This could mostly hurt usability in a system as **consistency and standards**[[1](https://www.nngroup.com/articles/consistency-and-standards/)] can be affected from having multiple bindings that lead to the same action. [Miller's law](https://en.wikipedia.org/wiki/Miller%27s_law) can be also helpful in this decision. Last but now least, studies[[1](https://www.researchgate.net/publication/12189991_When_Choice_is_Demotivating_Can_One_Desire_Too_Much_of_a_Good_Thing)][[2](https://www.nngroup.com/articles/minimize-cognitive-load/)] show that having too many options often leads to decision paralysis and frustration. 

🍋 🍋 🍋 🍋 

Like in https://github.com/gitpod-io/gitpod/pull/7715#discussion_r790696114, I'd suggest to leave this as is for now and come back to this if this becomes an issue unless @jldec has any strong feelings towards one specific direction. Cc @jankeromnes

Reply:
Thanks for the great references!

I ended up putting this to a poll, and for now Cmd/Ctrl+O won 😁 picking this as the current keybinding for now. We can always improve it later.

Reply:
FWIW, `Cmd/Ctrl+O` sounds great! Thanks for pushing this forward and going with one single keybinding for this iteration. 💯 

> We can always improve it later.

Yes! ➿